### ISSUE TITLE
Accessibility fixes

### ISSUE THREAD

**Issue by nfreear** [brainsum/cookieconsent#56]
Hi,

I'm identifying various Web accessibility (_accessibility to those with disabilities_).

I'd like to submit one or more fixes / pull requests for these — is this OK please?

Things that need fixing:

 * [ ] — Ensure all _buttons_ (including those that use `<div>`) are given a role of '_button_' — for screen reader/ non-visual users;
 * [ ] — Ensure all _buttons_ can be navigated to and activated with the keyboard ('_TAB_', shift-TAB, Enter keys) — keyboard users, including screen reader/ non-visual users;
 * [ ] — Ensure all _buttons_ have a suitable label/ name — non-visual users;
 * [ ] — Ensure the cookie-bar and cookie-modal have WAI-ARIA landmark regions, and labels;
 * [ ] — Ensure the cookie-modal behaves like proper modal (that is, links behind are inactive/ cannot be navigated to);
 * [ ] — _More ...?_

A number of the above are fairly straightforward fixes.

I'll let you know how I get on.

Yours,

Nick

**Comment by DanielNelsonCantarus** [comment]
Hi Nick, our team are looking to use this module and have also noticed the lack of accessibility features, have you made any progress on addressing any of the tasks you've raised here?
